### ISSUE TITLE
plugin adds machine specific settings to settings.json

### ISSUE THREAD

**Issue by rajsite** [AdobeXD/design-system-package-dsp#21]
We would like to have a `.vscode/settings.json` checked into the repo for a workspace to share between developers. An issue is that the Adobe XD extension appears to insert absolute paths into `.vscode/settings.json` instead of workspace relative paths:

```
{
    "xd.globalEditor": true,
    "xd.lastPackage": "/Users/<user>/Documents/GitRepositories/<path-to-package-in-repo>"
} 
```

Instead of a workspace relative path, ie:

```
{
    "xd.globalEditor": true,
    "xd.lastPackage": "<path-to-package-in-repo>"
} 
```

**Comment by jattasNI** [comment]
Agreed, the [docs for VS Code settings](https://code.visualstudio.com/docs/getstarted/settings) indicate that `vscode/settings.json` can be used for sharing settings between users and [the VS Code team uses that approach](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control), so writing user-specific content to that location doesn't seem quite right.

**Comment by kintz09** [comment]
I ran into this problem today as well. Our workspace settings are saved in the .vscode/settings.json file therefore we commit it to git.

I tried rewriting the path as a relative path, which other settings are using, but the XD extension didn't find the package so I'm assuming a relative path is not supported. I also tried removing the entry from workspace settings and moved it into user settings, but every time I open the XD extension it adds it back into the workspace settings?

Does anyone know a workaround?
